src/information-panel/information-panel.scss

```diff
@@ -187,3 +189,34 @@
     top: $iui-m * -0.5;
   }
 }
+
+/// Labels and inputs aligned using css grid
+@mixin iui-information-body-content {
+  .iui-input-label {
+    font-weight: normal;
+    @include themed {
+      color: t(iui-text-color-muted);
+    }
+  }
+
+  &:not(.iui-inline) {
+    // add bottom margin between pairs of label+input
+    > *:nth-child(even):not(:last-child) {
+      margin-bottom: 11px;
+    }
+  }
+
+  &.iui-inline {
+    display: grid;
+    row-gap: 11px;
+    // grid-template-columns: auto 1fr;
+    grid-template-columns: auto auto;
```

Comment:
I am struggling with this part.

`auto 1fr` will make the input take up the entire remaining space.
![image](https://user-images.githubusercontent.com/9084735/140421686-20e14796-af70-4214-a7a9-e435dfb0b01b.png)

`auto auto` will only make it take the least amount of space needed (even if the text inside the input is overflow).
![image](https://user-images.githubusercontent.com/9084735/140421647-150b5a7a-66f0-42f6-bd4c-50ee28cf6ea7.png)

@FlyersPh9 Previously you had this. Is there any logic or did you just find a random width that "looks nice"? 
![image](https://user-images.githubusercontent.com/9084735/140421524-814c9080-f5d1-4f30-827f-143934f98d23.png)

Reply:
Pretty sure I picked a value that worked well with the label lengths.

I like your first image as long as the 1st column is wide enough so none of the labels wrap to a 2nd line.

Reply:
No wrapping when there is space:
![image](https://user-images.githubusercontent.com/9084735/140565044-31290be5-6556-46d4-aa89-15b02de68177.png)

There is something going on in narrow widths though:
![image](https://user-images.githubusercontent.com/9084735/140565100-3f41525b-429e-45c1-adaa-e0553213c2fd.png)

Reply:
Aren't labels too wide in screenshot from description? It looks like it takes 60-70% of width. Is it really bad if some of them are multiline?
![image](https://user-images.githubusercontent.com/36186912/140695236-48735fe5-158e-4e83-b93d-8e25bbd2eaed.png)

Reply:
> No wrapping when there is space:
> There is some (w)rapping going on in narrow widths though:

I just checked this locally, what you have now looks good for the editable demo.  The labels should be on 1 line / not wrapping in our default width.  But if the user sets the width extremely narrow, it makes sense to wrap the labels.